tcms/static/js/testplan_actions.js

```diff
@@ -1644,115 +1644,6 @@ function onTestCaseTagFormSubmitClick(options) {
   };
 }
 
-function onTestCaseTagAddClick(options) {
```

Comment:
Also remove `onTestCaseTagFormSubmit` and `constructBatchtagProcessDialog` functions from this file.